sphinx_gallery/tests/testconfs/src/plot_1.py

```diff
@@ -1,7 +1,9 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
 """
```

Comment:
Ahh okay, I'll probably have to do something smarter then, I'll look into it more.